SparkPost/2web2ui
# Upgraded eslint to v5.15.2 to fix security vulnerabilities

_There is no ticket.  I am on support this week._

### What Changed

The following is part of the audit report.  I addressed high priority security vulnerabilities in a different PR.  This PR will focus exclusively on upgrading eslint from v5.0 to v5.15 to fix moderate level security vulnerabilities.

```
$ npm audit

                         === npm audit security report ===

  # Run  npm install --save-dev eslint@5.15.1  to resolve 2 vulnerabilities
  ┌───────────────┬──────────────────────────────────────────────────────────────┐
  │ Moderate      │ Prototype Pollution                                          │
  ├───────────────┼──────────────────────────────────────────────────────────────┤
  │ Package       │ lodash                                                       │
  ├───────────────┼──────────────────────────────────────────────────────────────┤
  │ Dependency of │ eslint [dev]                                                 │
  ├───────────────┼──────────────────────────────────────────────────────────────┤
  │ Path          │ eslint > inquirer > lodash                                   │
  ├───────────────┼──────────────────────────────────────────────────────────────┤
  │ More info     │ https://npmjs.com/advisories/782                             │
  └───────────────┴──────────────────────────────────────────────────────────────┘


  ┌───────────────┬──────────────────────────────────────────────────────────────┐
  │ Moderate      │ Prototype Pollution                                          │
  ├───────────────┼──────────────────────────────────────────────────────────────┤
  │ Package       │ lodash                                                       │
  ├───────────────┼──────────────────────────────────────────────────────────────┤
  │ Dependency of │ eslint [dev]                                                 │
  ├───────────────┼──────────────────────────────────────────────────────────────┤
  │ Path          │ eslint > lodash                                              │
  ├───────────────┼──────────────────────────────────────────────────────────────┤
  │ More info     │ https://npmjs.com/advisories/782                             │
  └───────────────┴──────────────────────────────────────────────────────────────┘

  found 4 vulnerabilities (1 low, 2 moderate, 1 high) in 112504 scanned packages
    run `npm audit fix` to fix 4 of them.
```

### How To Test

Just a bunch of fixes and chores in the [changelog](https://github.com/eslint/eslint/blob/master/CHANGELOG.md).  To confirm this is still working as expect, run `npm install && npm run lint` on master and the on this branch -- you should get the same number of errors.  We aren't going to fix those.  We incrementally fix as we touch a file.
